src/app/posts/post/post.component.html

```diff
@@ -0,0 +1,73 @@
+<form [formGroup]="postForm" *ngIf="postForm">
+  <div class="form-wrapper">
+    <div class="input-wrapper">
```

Comment:
You can create a shared component "form-input-group" and put the label + input there, so you call a single component here with different params - formControl, label text, etc.